dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -247,6 +233,8 @@
 
     private final JdbcTemplate jdbcTemplate;
 
+    private final JdbcEventCommentStore jdbcEventCommentStore;
```

Comment:
Is it possible to call this new store from the upstream service. This is fine, but in general, I prefer to avoid calling a dao from another dao.

Reply:
Sure, I'll check